packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -100,14 +101,11 @@ export function TransferPanel() {
     useState(false)
 
   const {
-    app: {
-      connectionState,
```

Comment:
i see that `connectionState` is not used in the whole app, let's nuke it from the code base